babel/messages/extract.py

```diff
@@ -236,9 +236,12 @@ def extract_from_file(method, filename, keywords=DEFAULT_KEYWORDS,
     :returns: list of tuples of the form ``(lineno, message, comments, context)``
     :rtype: list[tuple[int, str|tuple[str], list[str], str|None]
     """
-    with open(filename, 'rb') as fileobj:
-        return list(extract(method, fileobj, keywords, comment_tags, options,
-                            strip_comment_tags))
+    if method == 'ignore':
+        return []
+    else:
```

Comment:
Can you remove this `else:`? There's no point indenting the entirety of the (admittedly short) rest of the function.